src/commands/role.ts

```diff
@@ -0,0 +1,59 @@
+import {GuildMember, Message} from 'discord.js';
+import ICommand from '../library/iCommand';
+
+// Hack for implementing with static properties/methods
+let Role: ICommand;
+export default Role = class {
+
+  /* istanbul ignore next */
+  static get description(): string {
+    return 'Assigns a role to the user on request.';
+  }
+
+  public static execute(args: string[], msg: Message) {
+    const { channel } = msg;
+    const command = args[0];
+    const roleRequest = msg.guild.roles.find(r => r.name === args[1]); // Finds role based on input
```

Comment:
You may want to lowercase both sides of this condition.
I think role names are title cased, but I'm sure people will type in lowercase role names.